saleor/graphql/account/schema.py

```diff
@@ -296,25 +290,50 @@ class AccountMutations(graphene.ObjectType):
         )
     )
 
-    service_account_create = ServiceAccountCreate.Field()
-    service_account_update = ServiceAccountUpdate.Field()
-    service_account_delete = ServiceAccountDelete.Field()
+    service_account_create = ServiceAccountCreate.Field(
+        deprecation_reason=(
+            "Use the `appCreate` mutation instead. This field will be removed after "
+            "2020-07-31."
+        )
+    )
+    service_account_update = ServiceAccountUpdate.Field(
+        deprecation_reason=(
+            "Use the `appUpdate` mutation instead. This field will be removed after "
+            "2020-07-31."
+        )
+    )
+    service_account_delete = ServiceAccountDelete.Field(
+        deprecation_reason=(
+            "Use the `appDelete` mutation instead. This field will be removed after "
+            "2020-07-31."
```

Comment:
Souldn't we use a release number instead?

Reply:
We've changed approach to use date instead of version